Makefile

```diff
@@ -243,30 +244,43 @@ project/nmdc_schema_merged.yaml:
 #		--output project/prefixmap/nmdc.json \
 #	  	--mergeimports $@
 
-project/nmdc_materialized_patterns.yaml:
+#project/nmdc_materialized_patterns.yaml:
+#	$(RUN) gen-linkml \
+#		--format yaml \
+#		--materialize-patterns \
+#		--no-materialize-attributes \
+#		--output $@ $(SOURCE_SCHEMA_PATH)
+
+nmdc_schema/nmdc_materialized_patterns.yaml:
 	$(RUN) gen-linkml \
 		--format yaml \
 		--materialize-patterns \
 		--no-materialize-attributes \
 		--output $@ $(SOURCE_SCHEMA_PATH)
 
-project/nmdc_materialized_patterns.schema.json: project/nmdc_materialized_patterns.yaml
-	$(RUN) gen-json-schema \
-		--closed \
-		--include-range-class-descendants \
-		--top-class Database $< > $@
+#project/nmdc_materialized_patterns.schema.json: project/nmdc_materialized_patterns.yaml
+#	$(RUN) gen-json-schema \
+#		--closed \
+#		--include-range-class-descendants \
+#		--top-class Database $< > $@
 
 nmdc_schema/gold-to-mixs.sssom.tsv: sssom/gold-to-mixs.sssom.tsv nmdc_schema/nmdc_materialized_patterns.schema.json \
 nmdc_schema/nmdc_materialized_patterns.yaml nmdc_schema/nmdc_schema_merged.yaml
 	# just can't seem to tell pyproject.toml to bundle artifacts like these
 	#   so reverting to copying into the module
 	cp $< $@
 
-nmdc_schema/nmdc_materialized_patterns.schema.json: project/nmdc_materialized_patterns.schema.json
-	cp $< $@
+#nmdc_schema/nmdc_materialized_patterns.schema.json: project/nmdc_materialized_patterns.schema.json
+#	cp $< $@
 
-nmdc_schema/nmdc_materialized_patterns.yaml: project/nmdc_materialized_patterns.yaml
-	cp $< $@
+nmdc_schema/nmdc_materialized_patterns.schema.json: nmdc_schema/nmdc_materialized_patterns.yaml
+	$(RUN) gen-json-schema \
+		--closed \
+		--include-range-class-descendants \
+		--top-class Database $< > $@
+
+#nmdc_schema/nmdc_materialized_patterns.yaml: project/nmdc_materialized_patterns.yaml
+#	cp $< $@
```

Comment:
comment vs remove

Reply:
no, I should have deleted instead of commenting-out. I will do that in a  subsequent PR.